tests/jax/test_fused_attn.py

```diff
@@ -668,7 +706,7 @@ def check_dqkv(primitive, reference, pad):
         pytest.param(4, 128, 128, 16, 16, 64, jnp.bfloat16, id="4-128-128-16-16-64-BF16-SELF"),
         pytest.param(4, 128, 128, 16, 16, 64, jnp.float16, id="4-128-128-16-16-64-FP16-SELF"),
         pytest.param(2, 2048, 2048, 12, 12, 64, jnp.bfloat16, id="2-2048-2048-12-12-64-BF16-SELF"),
-        pytest.param(4, 512, 128, 16, 16, 64, jnp.bfloat16, id="4-512-128-16-16-64-BF16-CROSS"),
+        pytest.param(4, 128, 256, 16, 16, 64, jnp.bfloat16, id="4-128-256-16-16-64-BF16-CROSS"),
```

Comment:
Whats the reason for changing this test case? Is is that SWA does not work when Q < KV?

Reply:
cuDNN author confirmed that cuDNN does not support cross-attention SWA for `seqlen_q > seqlen_kv`. So this test case is to test cross-attention for `seqlen_q < seqlen_kv`, and the test case `2-2048-1024-12-12-64-BF16-CROSS` will be skipped.